.vscode/settings.json

```diff
@@ -0,0 +1,12 @@
+{
+    "rewrap.wrappingColumn": 80,
+    "editor.rulers": [80],
+    "markdownlint.config": {
+        "MD004": false,
```

Comment:
Will this list of exceptions need to be maintained indefinitely?

Reply:
We can update them when they are not required.

Reply:
I copied the file from the specs